ignite/pkg/cosmosgen/install_test.go

```diff
@@ -0,0 +1,93 @@
+package cosmosgen
```

Comment:
```suggestion
package cosmosgen_test
```

If so it would require other changes in this file.